radicle-cli/src/commands/comment.rs

```diff
@@ -75,7 +75,6 @@ impl Args for Options {
 
                 // Common.
                 Long("help") => return Err(Error::Help.into()),
-
```

Comment:
Please don't change the formatting of existing code unless something's wrong with it :smile: